### DISCUSSION TITLE
Code browsing (jump to definition) on mobile?

### DISCUSSION THREAD

**Discussion by yang** [community/community#22749] [New to GitHub]
<p>I seem to only be able to use semantic code browsing features (like jump to definition) on desktop. Are these features disabled on the mobile browser? I couldn’t seem to get it to work from tapping around.</p>

**Comment by Jme7** [comment, marked as answer]
<p>if it only works on browser, then you can opt-out of mobile pages and just use desktop pages in settings.</p>
<p>Sorry if this is a bad solution, this is all I know of :frowning:</p>
